redi-connect-front/src/pages/front/reset-password/SetNewPassword.tsx

```diff
@@ -123,7 +122,6 @@ export const SetNewPassword = (props: RouteComponentProps<RouteParams>) => {
   ) => {
     try {
       await setPassword(values.password)
-      await giveGdprConsent()
       await activateUser()
```

Comment:
Please also remove the call to `activateUser()`. The only use case for turning an inactive user into an active one is when an administrator (e.g. Isabelle) does so via the admin panel.